io/io/test/TBufferMerger.cxx

```diff
@@ -118,48 +118,6 @@ TEST(TBufferMerger, ParallelTreeFill)
    EXPECT_TRUE(FileExists("tbuffermerger_parallel.root"));
 }
 
-TEST(TBufferMerger, AutoSave)
```

Comment:
Why are we removing this test?

Reply:
Because the `fAutoSave` setting of `TBufferMerger` was only relevant for pushing buffers into the queue:
```c++
   if (fBuffered > fAutoSave)
      Merge();
```
After the simplifications, we always merge from the live `TTree`.